website/source/docs/guides/production.html.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: "docs"
+page_title: "Production Hardening"
+sidebar_current: "docs-guides-prod"
+description: |-
+  Production hardening of a Vault setup.
+---
+
+# Production Hardening
+
+This guide provides guidance on best practices for a production hardened deployment of Vault.
+The recommendations are based on the [security model](/docs/internals/security.html) and focus on defense in depth.
+
+* **End-to-End TLS**. Vault should always be used with TLS in production. If intermediate load balancers or reverse proxies are used to front Vault, they should _not_ terminate TLS. This way traffic is always encrypted in transit to Vault and minimizes risks introduced by intermediate layers.
+
+* **Single Tenancy**. Vault should be run as a single tenant application on a machine. This reduces the risk that another machine running on the same machine is compromised and can interact with Vault. Similarly, running on bare metal should be preferred to a VM, and a VM preferred to a container. This reduces the surface area introduced by additional layers of abstraction and other tenants of the hardware.
+
+* **Firewall traffic**. Vault listens on well known ports, so a local firewall should be used to restrict all incoming and outgoing traffic to Vault and essential system services like NTP.
+
+* **Disable SSH / Remote Desktop**. When running a Vault as a single tenant application, there should be no reason users needs access to the machine. Users and Applications should interact with Vault entirely over the network. Logs and telemetry should be shipped out for debugging.
```

Comment:
> there should be no reason users needs access to the machine

consider

users should never access the machine directly. Instead, they should access Vault through its API over the network. Use a centralized logging and telemetry solution for debugging.